### PR TITLE
Updated the main email article to Symfony 4/Flex

### PR DESCRIPTION
**Pull request by javiereguiluz**

### CODE REVIEW

File: email.rst [hunk 1]
```diff
@@ -4,102 +4,57 @@
 How to Send an Email
 ====================
 
-Sending emails is a classic task for any web application and one that has
-special complications and potential pitfalls. Instead of recreating the wheel,
-one solution to send emails is to use the SwiftmailerBundle, which leverages
-the power of the `Swift Mailer`_ library. This bundle comes with the Symfony
-Standard Edition.
+Sending emails is a common task for any web application and one that has
+special complications and potential pitfalls. Symfony provides a mailer feature
```
Comment (weaverryan) [inline email.rst]: I think the first sentence is crap... and it's not *your* change - I think the old sentence (I probably wrote it?) is crap. Let's remove it entirely.

File: email.rst [hunk 2]
```diff
@@ -4,102 +4,57 @@
...
-        swiftmailer:
-            transport: '%mailer_transport%'
-            host:      '%mailer_host%'
-            username:  '%mailer_user%'
-            password:  '%mailer_password%'
+    $ composer require mailer
 
-    .. code-block:: xml
+If your application doesn't use Symfony Flex, follow the installation
+instructions of the `SwiftMailerBundle`_.
```
Comment (weaverryan) [inline email.rst]: instructions on SwiftMailerBundle...

File: email.rst [hunk 3]
```diff
@@ -4,102 +4,57 @@
...
-            'username'   => "%mailer_user%",
-            'password'   => "%mailer_password%",
-        ));
+Configuration
+-------------
 
-These values (e.g. ``%mailer_transport%``), are reading from the parameters
-that are set in the :ref:`parameters.yml <config-parameters.yml>` file. You
-can modify the values in that file, or set the values directly here.
+The ``config/packages/swiftmailer.yaml`` file created when installing the mailer
```
Comment (weaverryan) [inline email.rst]: ... file that's created...

File: email.rst [hunk 4]
```diff
@@ -4,102 +4,57 @@
...
+provides all the initial config needed to make it work, except the parameters
```
Comment (weaverryan) [inline email.rst]: ... needed to send emails, except your mail server connection details.

File: email.rst [hunk 5]
```diff
@@ -4,102 +4,57 @@
...
 
-* ``transport`` (``smtp``, ``mail``, ``sendmail``, or ``gmail``)
-* ``username``
-* ``password``
-* ``host``
-* ``port``
-* ``encryption`` (``tls``, or ``ssl``)
-* ``auth_mode`` (``plain``, ``login``, or ``cram-md5``)
-* ``spool``
+    # use this to disable the email delivery
```
Comment (weaverryan) [inline email.rst]: > ... to disable email delivery

File: email.rst [hunk 6]
```diff
@@ -4,102 +4,57 @@
...
+    MAILER_URL=null://localhost
 
-  * ``type`` (how to queue the messages, ``file`` or ``memory`` is supported, see :doc:`/email/spool`)
-  * ``path`` (where to store the messages)
-* ``delivery_addresses`` (an array of email addresses where to send ALL emails)
-* ``disable_delivery`` (set to true to disable delivery completely)
+    # use this to send emails via Gmail (don't use this in production)
+    MAILER_URL=gmail://username:password@localhost
```
Comment (weaverryan) [inline email.rst]: Can we / should we delete the `gmail.rst` article now?

File: email.rst [hunk 7]
```diff
@@ -4,102 +4,57 @@
...
 
-.. caution::
+    # use this to configure a traditional SMTP server
+    MAILER_URL=smtp://localhost:25?encryption=ssl&auth_mode=login&username=&password=
 
-    Starting from SwiftMailer 5.4.5, the ``mail`` transport is deprecated
-    and will be removed in version 6. Consider using another transport like
-    ``smtp``, ``sendmail`` or ``gmail``.
+Refer to the :doc:`SwiftMailer configuration reference </reference/configuration/swiftmailer>`
```
Comment (weaverryan) [inline email.rst]: Is there an easy example we could show for Mandrill, SendGrid or Amazon SES? I believe you basically use `smtp`, but point to their servers. But that might not obvious without an example.

And if we show that here, could we delete `cloud.rst`?

**Review by weaverryan — CHANGES_REQUESTED** [review]

**Comment by weaverryan** [comment]
Status: Needs Work

File: email.rst (hunk 6 above)
Comment (javiereguiluz) [reply email.rst]: Yes! I'm proposing to remove two email articles: #8668.

File: email.rst (hunk 7 above)
Comment (javiereguiluz) [reply email.rst]: Yes, let's do that when we remove the other articles.

**Pull request merged**